README.adoc

```diff
@@ -108,6 +108,35 @@ List<Product> found = products.findByNameLike(searchPattern, Order.by(
                                               _Product.id.asc()));
 ----
 
+== Jakarta Data Query Language (JDQL)
+
+Jakarta Data introduces the Jakarta Data Query Language (JDQL), a streamlined query language designed to specify the semantics of query methods within Jakarta Data repositories. Utilizing the `@Query` annotation, JDQL allows developers to define queries straightforwardly and robustly.
+
+JDQL is conceptualized as a subset of the Jakarta Persistence Query Language (JPQL). It inherits its syntax and functionality while being specifically tailored to accommodate the broad spectrum of data storage technologies supported by Jakarta Data. This design approach ensures that JDQL retains the expressiveness of SQL and JPQL yet simplifies its implementation across diverse data stores.
```

Comment:
Not sure it's correct to say that JDQL retains the "expressiveness" of SQL/JPQL, if you can express much less with it.

Might be better to say "familiarity" or "usability" or "understandability" or something like that.